### ISSUE TITLE
Fix a11y issues: Links must have discernible text

### ISSUE THREAD

**Issue by christiemolloy** [patternfly/patternfly-org#1352]
On multiple pages we are getting the error "Links must have discernible text". This is a serious violation.

The problem is that anchor links do not have discernible text, we could add an aria-label to them to fix this. Here is the documentation of the problem. https://dequeuniversity.com/rules/axe/3.2/link-name?application=AxeChrome

The problem is that for every anchor link that we have, we have an a11y violation so its very large. In the `<a>` tag is also an `href` that has the name of the anchored element, I think that if we write a script we could use the name for the `href` for the `aria-label` attribute.

@redallen since this issue is so large, would you be able to create a script to fix all of them, I can work with you on this.

**Comment by jessiehuff** [comment]
This got cleaned up [here](https://github.com/patternfly/patternfly-org/pull/1588/files#diff-97da1b50385b66775aff7e7c14e2452fR32). 🙂  So the rationale behind this is that the functionality of these header links isn't as useful for screen reader users and likely would add additional noise by doubling the amount of links they have to go through. It seemed to make more sense to make it hidden. However, I can see the other side of the argument so I'm open to discussing this one again. If there's any strong opinions on this, I'm happy to reopen or create a new issue!
